packages/app-check/src/api.ts

```diff
@@ -65,6 +67,21 @@ export function initializeAppCheck(
         options.isTokenAutoRefreshEnabled &&
       initialOptions.provider.isEqual(options.provider)
     ) {
+      // Log a warning if `initializeAppCheck()` is called after the first time
+      // if this app is still in debug mode, and show the token.
+      // If it's the first time, `initializeDebugMode()` already logs a message.
+      if (isDebugMode()) {
+        logger.warn(
+          `App Check is in debug mode. To turn off debug mode, unset ` +
+            `the global variable FIREBASE_APPCHECK_DEBUG_TOKEN and ` +
+            `restart the app.`
+        );
+        // Make this a separate console statement so user will at least have the
+        // first message if the token promise doesn't resolve in time.
+        void getDebugToken().then(token =>
+          logger.warn(`Debug token is ${token}.`)
+        );
```

Comment:
We should print the debug token every time when `initializeAppCheck` is called if it's in the debug mode, not just the second time. For example, print the token after the page is refreshed.

Reply:
Ok, can do, just FYI, if it's the first call to initializeApp(), the user will see
`App Check debug token: ${newToken}. You will need to add it to your app's App Check settings in the Firebase console for it to work`
And then they will also see these 2 messages right after that. Is that a bit clunky?

I guess the exception is if they have 2 FirebaseApps, then they'll only see all 3 messages when calling initializeApp for the first FirebaseApp.

Reply:
`logger.warn(`Debug token is ${token}.`)` is not necessary, if we already print `App Check debug token: ${newToken}. You will need to add it to your app's App Check settings in the Firebase console for it to work`.

Reply:
so no matter whether it's the first or nth time, always show the same message.

Reply:
Ok, done, see if that looks acceptable now.